### ISSUE TITLE
Question: Component styles

### ISSUE THREAD

**Issue by d3viant0ne** [ngParty/ng-metadata#63]
First off.....This is really cool, it's been a joy to work with thus far.

I did have one question when looking at back porting an existing Angular2 application to 1.5.x using ng-metadata. Component styles ala `styleUrls:`

Now having said that, I understand the technical challenges with implementing that in a 1.5.x environment, I was more curious if it was on the roadmap or more trouble than it's worth.

**Comment by Hotell** [comment]
Hey @d3viant0ne thanks for kind words!

I was thinking about doing that, but it would be really an overkill... so I've stepped away.

Anyway we have other tools for that. You can do exactly the same with webpack in both angular 2 or angular 1 with ngMetadata like this https://github.com/ngParty/Angular1-scaffold/pull/9/commits/add307f84b10427d081129382fc090bb6ecc5e6b

> this should be also able with SystemJS if that's your bundler of choice

only difference is that you have to manually prefix your styles with the component selector to have proper "encapsulation" 

so in the example above it would look like this:
```scss
app{
  background-color: black;
  color: navajowhite;
 
  h1{ }
 .my-paragraph{ }
 // etc

}
```

P.S.: I highly recommend our scaffold/starter for building migration ready angular 1 apps if you just can't start with angular 2 right away https://github.com/ngParty/Angular1-scaffold

cheers!

**Comment by d3viant0ne** [comment]
@Hotell - Yeah, I was just thinking about it from a feature parity perspective and wanted to ask before I went another direction, plenty of ways to build in encapsulation.

 Actually I'm in the middle of porting Minko Gechev's angular2-seed (i'm one of the collabs) to support a 1.5 application to try and simplify the transition for those who are planning on using the 2.0 version of the tooling but still need to build with 1.5. 

So if you end up getting an influx of silly questions in a few weeks when my little franken-starter is moved over to Minko's org, that's probably going to be my fault.

And for reference, I've poked around the scaffold you mentioned but alas I just can't learn to love Webpack. Amazing piece of engineering but just too much "magic" for my taste and I usually try to stay inline with the framework i'm using 

i.e. angular2 / systemjs

Thanks for the quick reply, i'm sure it won't be the last question I ask in the next few days.
